Libraries/Text/Text.js

```diff
@@ -154,7 +158,7 @@ const Text = React.createClass({
                 }
               }
               this.touchableHandleActivePressIn = () => {
-                if (this.props.suppressHighlighting || !this.props.onPress) {
+                if (this.props.suppressHighlighting || !this.props.onPress && !this.props.onLongPress) {
```

Comment:
Sounds reasonable to me. I am thinking it could just be `const hasPressHandlers` instead. What you think?